check_rbl.ini

```diff
@@ -34,8 +66,120 @@ server=forbidden.icm.edu.pl
 server=hil.habeas.com
 server=rbl.schulte.org
 server=sbl-xbl.spamhaus.org
+server=b.barracudacentral.org
+server=cdl.anti-spam.org.cn
+server=dnsbl.antispam.or.id
+server=dnsbl.inps.de
+server=drone.abuse.ch
+server=dsn.rfc-ignorant.org
+server=dul.dnsbl.sorbs.net
+server=http.dnsbl.sorbs.net
+server=l1.spews.dnsbl.sorbs.net
+server=l2.spews.dnsbl.sorbs.net
+server=misc.dnsbl.sorbs.net
+server=postmaster.rfc-ignorant.org
+server=rbl.spamlab.com
+server=rbl.suresupport.com
+server=relays.bl.kunden.de
+server=short.rbl.jp
+server=smtp.dnsbl.sorbs.net
+server=socks.dnsbl.sorbs.net
+server=spam.abuse.ch
+server=spamrbl.imp.ch
+server=tr.countries.nerd.dk
+server=unsure.nether.net
+server=virbl.bit.nl
+server=virus.rbl.jp
+server=web.dnsbl.sorbs.net
+server=whois.rfc-ignorant.org
+server=wormrbl.imp.ch
+server=zen.spamhaus.org
+server=zombie.dnsbl.sorbs.net
+server=blackholes.five-ten-sg.com
+server=blacklist.woody.ch
+server=bl.emailbasura.org
+server=bogons.cymru.com
+server=combined.abuse.ch
+server=duinv.aupads.org
+server=dynip.rothen.com
+server=ohps.dnsbl.net.au
+server=omrs.dnsbl.net.au
+server=orvedb.aupads.org
+server=osps.dnsbl.net.au
+server=osrs.dnsbl.net.au
+server=owfs.dnsbl.net.au
+server=owps.dnsbl.net.au
+server=probes.dnsbl.net.au
+server=proxy.bl.gweep.ca
+server=proxy.block.transip.nl
+server=rbl.inter.net
+server=rbl.megarbl.net
+server=rdts.dnsbl.net.au
+server=relays.bl.gweep.ca
+server=residential.block.transip.nl
+server=ricn.dnsbl.net.au
+server=rmst.dnsbl.net.au
+server=spamlist.or.kr
+server=t3direct.dnsbl.net.au
+server=ubl.lashback.com
+server=all.s5h.net
+server=b.barracudacentral.org
+server=bl.emailbasura.org
+server=bl.spamcannibal.org
+server=bl.spamcop.net
+server=blacklist.woody.ch
```

Comment:
duplicate (may contain more?)

Reply:
Thanks, you are correct. It would appear so. 
Let me fix this.